extensions/2.0/Vendor/GRIFFEL_bim_data/schema/GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,105 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "GRIFFEL_bim_data Serializable Object",
+    "type": "object",
+    "description": "Contains same properties as embedded version plus properties to node map. Used only when data is saved in a separate file.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "nodeProperties": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "description": "Node properties and type.",
+                "properties": {
+                    "node": {
+                        "allOf": [ { "$ref": "glTFid.schema.json" } ],
+                        "description": "Index of the node in glTF file."
```

Comment:
What happens when two elements of the `nodeProperties` array point to the same node?

Reply:
I'm adding lines to the _Readme.md_:
>If embedded data duplicates properties for a node it is up to Importer: ignore duplicates or apply them all.